src/components/molecules/card/CardFooter.tsx

```diff
@@ -20,14 +21,25 @@ const useStyles = makeStyles({
     alignItems: 'flex-end',
     justifyContent: 'flex-end',
   },
+  spaceSpan: {
+    marginLeft: '4px',
+    marginRight: '4px',
+  },
 });
 
 const CardFooter: React.FC<IProps> = ({ dateComment }) => {
+  const { t } = useTranslation();
   const classes = useStyles();
-  const dateRange = dateComment?.date_range?.join('-') ?? ''
+  const lastDate = new Date(dateComment.last_update);
+  const lastDateString = dateComment.last_update? `${lastDate.getDate()}/${lastDate.getMonth() + 1}/${lastDate.getFullYear()}`: ""
```

Comment:
do not format yourself - use `dateFormat`

Reply:
ok, shall i install the dateFormat package? 
I couldn't import it & didn't find it in package.json .